azext_edge/edge/providers/support/akri.py

```diff
@@ -42,6 +49,23 @@ def fetch_pods(since_seconds: int = DAY_IN_SECONDS):
             since_seconds=since_seconds,
         )
     )
+
+    pod_name_label = AKRI_NAME_LABEL.replace("0", f"{AKRI_AGENT_LABEL}, {AKRI_WEBHOOK_LABEL}")
+    processed.extend(
+        process_v1_pods(
+            resource_api=AKRI_API_V0,
+            label_selector=pod_name_label,
+            since_seconds=since_seconds,
+        )
+    )
+
+    processed.extend(
```

Comment:
for these - can you combine the labels to minimize code that needs to be removed?

so here you would call process_v1_pods once and just have the labels get everything (with comments of what you want to remove if needed)